src/renderers/shaders/ShaderChunk/skinning_vertex.glsl

```diff
@@ -7,6 +7,7 @@
 	skinned += boneMatY * skinVertex * skinWeight.y;
 	skinned += boneMatZ * skinVertex * skinWeight.z;
 	skinned += boneMatW * skinVertex * skinWeight.w;
-	skinned  = bindMatrixInverse * skinned;
+
+	transformed = ( bindMatrixInverse * skinned ).xyz;
```

Comment:
> If that's the case, then you can drop the w-component since it will always be 1.

Not if the weights are all zero, for example. Of course, they shouldn't be.

You are assuming the weights are normalized. They are normalized in the constructor, so I think you are OK here. I still think doing an experiment to verify the value of `.w` would be a good idea.